run/helloworld/README.md

```diff
@@ -33,9 +41,6 @@ _Note: you may need to install `pytest` using `pip install pytest`._
 # Set an environment variable with your GCP Project ID
 export GOOGLE_CLOUD_PROJECT=<PROJECT_ID>
 
-# Submit a build using Google Cloud Build
-gcloud builds submit --tag gcr.io/${GOOGLE_CLOUD_PROJECT}/helloworld
-
 # Deploy to Cloud Run
 gcloud run deploy helloworld \
 --image gcr.io/${GOOGLE_CLOUD_PROJECT}/helloworld
```

Comment:
Thanks for this update! Can you change this to use deploy from source?

Reply:
yup, updated!